KANI_AUDIT_REPORT.md

```
@@ -0,0 +1,110 @@
+When `execute_trade` applies trade PnL that reduces an account's PnL below its `reserved_pnl`,
+this invariant is violated. The `reserved_pnl` field (part of the warmup mechanism) is not
+adjusted downward when trade PnL brings total PnL below the reserved amount.
+
+### Reproduction
+```bash
+cd percolator
+cargo kani --harness fast_valid_preserved_by_execute_trade --tests
+```
+
+Output:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)